executor/src/lib.rs

```diff
@@ -900,7 +906,10 @@ mod tests {
 		.unwrap()
 		.into_encoded();
 		let r = ApplyExtrinsicResult::decode(&mut &r[..]).unwrap();
-		assert_eq!(r, Err(InvalidTransaction::Payment.into()));
+		assert_eq!(
+			r,
+			Err(TransactionValidityError::Invalid(InvalidTransaction::Custom(198)))
```

Comment:
Is it possible to import the consts here so that:
`198` -> `INSUFFICIENT_SELLER_CORE_ASSET_BALANCE`?

Reply:
Good point